Tutorials/G-CNN Honeycomb.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<p>This should be cleaned up. It's erroring (or you did not rerun the code)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
hmm i thought I had ran everything, I'll see what's going on

Reply:
<p>Ahh i see, yeah I think this is from running things in the wrong order. I will run the notebook on the GPU and upload that version</p>